jupyter-notebooks/Data-API/search_and_preview_quickstart.ipynb

```diff
@@ -0,0 +1,496 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6db765fd-52c6-42d1-853c-4958a42f7099",
+   "metadata": {},
+   "source": [
+    "## Search & preview some imagery of farmland near Stockton, CA.\n",
+    "\n",
+    "## Motivation\n",
+    "In many workflows, previewing Planet imagery provides immediate value without the need for full order activation or processing. Similar to the web app experience of Planet Explorer, you can preview imagery directly from Data API search results, enabling pre-filtering and informed decisions before placing an order. This guide demonstrates how to leverage Planet’s preview tiles and the Data API to quickly visualize search results on an interactive map, facilitating more efficient data review and planning.\n",
+    "\n",
+    "\n",
+    "By the end of this guide, you will have an interactive map showing Planet imagery tiles and a clearly demarcated search AOI to assist with the imagery review.\n",
```

Comment:
```suggestion
    "This guide demonstrates how to leverage Planet’s preview tiles and the Data API to visualize search results on an interactive map with a clearly demarcated search AOI for quick, efficient imagery review and planning.\n",
```

Reply:
I'm not sure how we want to phrase this, but it also might be worth adding that this is also cheaper than the activation notebook.